packages/dx-react-scheduler/src/plugins/all-day-panel.test.tsx

```diff
@@ -57,6 +58,22 @@ describe('AllDayPanel', () => {
     jest.resetAllMocks();
   });
 
+  describe('Getters', () => {
+    it('should provide allDayElementsMeta getter', () => {
+      const tree = mount((
+        <PluginHost>
+          {pluginDepsToComponents(defaultDeps)}
+          <AllDayPanel
+            {...defaultProps}
+          />
+        </PluginHost>
+      ));
+
+      expect(getComputedState(tree).allDayElementsMeta)
```

Comment:
We should add test against allDayElementsMeta that checks how this getter is computed